candidate-standard/abstract_tests/collections/ATS_rc-md-items.adoc

```diff
@@ -6,8 +6,10 @@
 ^|Requirement |<<req_collections_rc-md-items,/req/collections/rc-md-items>>
 ^|Test Method |. Verify that there is an entry in the `collections` array of the Collections Metadata for each feature collection provided by the API.
```

Comment:
Should this be "feature collection" here?

Reply:
Yes, I think so. That is what 'Items' is for. Isn't this what you wanted to retrieve observations?

Reply:
created issue #90